impeller/typographer/backends/skia/text_render_context_skia.cc

```diff
@@ -139,7 +141,9 @@ ISize OptimumAtlasSizeForFontGlyphPairs(
     const FontGlyphPair::Set& pairs,
     std::vector<Rect>& glyph_positions,
     const std::shared_ptr<GlyphAtlasContext>& atlas_context) {
-  static constexpr auto kMinAtlasSize = 8u;
+  // This size needs to be above the minimum required aligment for linear
+  // textures. This is 256 for older intel macs and decreases on iOS devices.
+  static constexpr auto kMinAtlasSize = 256u;
```

Comment:
nit: Look this up based on caps?

Reply:
I think it might actually be better to keep a larger minimum atlas size since we reuse the texture across frames now, but only if the size is unchanged. The memory hit is fairly minor

Reply:
Yes, but what happens if we run on some backend where the minimum size is even larger?

Reply:
it only matters for metal

Reply:
I suppose its possible that a future iOS or macOS versions requires a larger minimum alignment. I'll update this to compute the exact minimum alignment.

That said, separately I think we should use a much larger minimum texture size